src/app/common-ui/Popup/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styles from './style.scss';
+import PropTypes from 'prop-types';
+
+import CloseButton from '../../../assets/images/close-button.png';
+
+function Popup(props) {
+  const { onClose, title, content, buttonText, variant, onButtonClick } = props;
+
+  return (
+    <div className={styles.overlay}>
+      <div className={styles.container}>
+        <div className={styles.titleBar}>
+          <div
+            className={`${styles.title} ${
+              variant === 'success' ? styles.titleColorSuccess : styles.titleColorFailed
+            }`}
+          >
+            {title}
+          </div>
+          <div className={styles.closeImg} onClick={onClose}>
+            <img src={CloseButton} alt="close" />
+          </div>
+        </div>
+        <div className={styles.content}>
+          <div>{content}</div>
+        </div>
+        <button className={styles.closeBtn} type="submit" onClick={onButtonClick}>
+          {buttonText}
+        </button>
+      </div>
+    </div>
+  );
+}
+
+Popup.propTypes = {
+  onClose: PropTypes.func,
+  content: PropTypes.string,
+  title: PropTypes.string,
+  buttonText: PropTypes.string,
+  variant: PropTypes.string,
+  onButtonClick: PropTypes.func,
+};
+
+Popup.defaultProps = {
+  onClose: null,
+  content: '',
+  title: '',
+  buttonText: '',
+  onButtonClick: null,
+  variant: '0',
```

Comment:
Because currently you only compare this props with `success` so lets change this default to something like `fail` or `warning`
`'0'` is not descriptive is enough for this props `variant`